cmd/bundle/list.go

```diff
@@ -36,39 +34,20 @@ func listBundlesCommand() *cobra.Command {
 }
 
 func listBundles(config *restclient.Config, ns string) error {
-	kubeClient := kubernetes.NewForConfigOrDie(config)
-	corev1 := kubeClient.CoreV1()
-	secretsApi := corev1.Secrets(ns)
-	opts := metav1.ListOptions{
-		LabelSelector: "managed-by=arlon,arlon-type=config-bundle",
-	}
-	secrets, err := secretsApi.List(context.Background(), opts)
+	bundles, err := bundle.List(config, ns)
 	if err != nil {
-		return fmt.Errorf("failed to list secrets: %s", err)
-	}
-	if len(secrets.Items) == 0 {
-		fmt.Println("no bundles found")
-		return nil
+		if errors.Is(err, bundle.ErrNoBundles) {
```

Comment:
Absence of bundles should not be an error. See related comment in the function.

Reply:
done